mods/d2k/rules/defaults.yaml

```
@@ -153,16 +153,24 @@
 		AttackMoveCondition: attack-move
 		AssaultMoveCondition: assault-move
 
+^AutoTargetVehicleAssaultMove:
+	Inherits: ^AutoTargetGroundAssaultMove
+	AutoTargetPriority@VEHICLES:
+		ValidTargets: Vehicle
+		InvalidTargets: Infantry, Structure, Defense
+		RequiresCondition: bot-owned && (attack-move || assault-move)
```

```suggestion
		RequiresCondition: bot-owned
```

Shouldn't this priority be universally applied? If this priority doesn't return any targets it should move onto the next priority correct?

---

`bot-owned && (attack-move || assault-move)` is used elsewhere in yaml. I was trying to stay with convention.

---

It's not used everywhere. Only where it is needed

---

I think this is a leftover from my initial attempt where I made player controlled Deviators smart, too.